replication.go

```diff
@@ -1185,6 +1185,10 @@ func (cm *ContentManager) SetDataTransferStartedOrFinished(ctx context.Context,
 		return err
 	}
 
+	if chanst == nil {
```

Comment:
@jlogelin @alvin-reyes this should not be the case, this state should exist. I am working on a task to always have a remote state, the current implementation is not sufficient